articles/api-management/api-management-using-with-vnet.md

```diff
@@ -78,6 +78,7 @@ The API Management service depends on several Azure services. When API Managemen
 
 > [!IMPORTANT]
 > If you plan to use a custom DNS server(s) for the VNet, set it up **before** deploying an API Management service into it. Otherwise, you'll need to update the API Management service each time you change the DNS Server(s) by running the [Apply Network Configuration Operation](/rest/api/apimanagement/current-ga/api-management-service/apply-network-configuration-updates).
+> The process of applying Network Configuration might take 15 to 45 minutes or more. The Developer SKU has downtime during the process. The Basic and higher SKUs don't have downtime during the process.
```

Comment:
Recommend that you remove this note here and add info about downtime at line 52 which already mentions the 15 to 45 minutes. Suggest the following at line 52: "It can take 15 to 45 minutes to update the API Management instance. The Developer tier has downtime during the process. The Premium tier doesn't have downtime."
I recommend not mentioning Basic and Standard tiers since those don't support VNet currently.

Reply:
hi @dlepow , Thanks for your suggestion.
Updated the changes as recommended.

Also Updated the doc  "api-management-using-with-internal-vnet.md"
https://github.com/MicrosoftDocs/azure-docs/pull/89325